All_home_works/hw2/task1.py

```diff
@@ -30,6 +30,14 @@ def sort_counter(self):
         )
         return list(sorted_counter)
 
+    @property
+    def rarest_element(self):
+        return min(self.sort_counter, key=lambda item: item[1])[0]
```

Comment:
That's why the test changed

Reply:
I have already switched my context.  This is just a line for me, can you give me a short description of what was before and why you have changed the result of that function execution. Was it an issue of something else?

Reply:
What is the complexity of this line? Can we do better?
@E1-Bart0 E1-Bart0 2 days ago Author Owner

I remade for O(n). And because there are some rarest chars new method gives us different from the previous method char

Reply:
> gives us different from the previous method char

Why does it give a different result?

Reply:
![image](https://user-images.githubusercontent.com/70700249/116989164-0c0da180-acda-11eb-84a8-70196d40e8f6.png)

Reply:
Can you remove reverse without breaking another code?

Reply:
What is the complexity of this line? Can we do better?

Reply:
Now I'm using the min function(), not sorted. So the complexity is O(n) and we can't do better in this structure.
Maybe a little bit better to use: min(self.counter, key=self.counter.get)